### ISSUE TITLE
NTR:Bilateral frontal polymicrogyria

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4562]
label:Bilateral frontal polymicrogyria (synonym: ) ID NA parent Frontal polymicrogyria def Symmetric polymicrogyria with anterior > posterior gradient extending from frontal poles posteriorly to precentral gyrus and inferiorly to frontal operculum [20301504] COST:neuromig,

**Comment by pnrobinson** [comment]
This should be coded using the modifier Bilateral
https://hpo.jax.org/app/browse/term/HP:0012832
